src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -24,8 +24,8 @@
         "symfony/config": "^3.4.31|^4.3.4",
         "symfony/debug": "~2.8|~3.0|~4.0",
         "symfony/event-dispatcher": "~3.4|~4.0",
-        "symfony/http-foundation": "^3.3.11|~4.0",
-        "symfony/http-kernel": "~3.4|~4.0",
+        "symfony/http-foundation": "^3.4.13|~4.0",
```

Comment:
https://github.com/symfony/symfony/commit/26fc4e683f96b51237fb206f3afbf93525adc49c is required